config/config.exs

```diff
@@ -78,6 +78,10 @@ config :prometheus, Dev3.PipelineInstrumenter,
 # as well as ...
 config :dev3, Dev3.Repo,
   loggers: [Dev3.RepoInstrumenter] # and maybe Ecto.LogEntry? Up to you
+
+config :dev3, max_requests: 5
```

Comment:
Can you update this to a more concise form ? Example : 

```elixir
  config :dev3, :rate_limiter,
     max_requests: 5,
     ignore: 7,
     interval_seconds: 60
```

It will then be available like this `Application.get_env(:dev3, :rate_limiter)[:max_requests]`